include/dlaf/matrix/print_csv.h

```diff
@@ -7,10 +7,16 @@
 // Please, refer to the LICENSE file in the root directory.
 // SPDX-License-Identifier: BSD-3-Clause
 //
+
+#include <iomanip>
+#include <iostream>
+
 #pragma once
 
 /// @file
 
+#include <string>
```

Comment:
Please be consistent with where you place the includes.

Reply:
Hi Mikael, I'm trying to fix my settings on VsCode. Because when I'm saving my changes my IDE automatically indent and makes some modifications.